horizon/enforcer/api.py

```diff
@@ -490,19 +504,26 @@ async def user_tenants(
         query: UserTenantsQuery,
         x_permit_sdk_language: Optional[str] = Depends(notify_seen_sdk),
     ):
-        response = await _is_allowed(query, request, USER_TENANTS_POLICY_PACKAGE)
-        log_query_result(query, response)
-        try:
-            raw_result = json.loads(response.body).get("result", {})
+        def parse_func(result: dict | list) -> dict | list:
             if isinstance(raw_result, dict):
-                tenants = raw_result.get("tenants", {})
+                tenants = result.get("tenants", [])
             elif isinstance(raw_result, list):
-                tenants = raw_result
+                tenants = result
             else:
                 raise TypeError(
                     f"Expected raw result to be dict or list, got {type(raw_result)}"
                 )
-            result = parse_obj_as(UserTenantsResult, tenants)
+            return tenants
+        raw_result = await conditional_is_allowed(
```

Comment:
Why not call `_is_allowed_data_manager` directly here instead?

Reply:
because the old system also has support for get user tenants, the conditional basically determines which one should be chosen, the new datasync or the old one